tests/end-to-end/director/application_api_test.go

```diff
@@ -405,41 +405,64 @@ func TestUpdateApplicationParts(t *testing.T) {
 
 	t.Run("manage webhooks", func(t *testing.T) {
 		// add
-		webhookInStr, err := tc.graphqlizer.ApplicationWebhookInputToGQL(&graphql.ApplicationWebhookInput{
+		webhookInStr, err := tc.graphqlizer.WebhookInputToGQL(&graphql.WebhookInput{
 			URL:  "new-webhook",
 			Type: graphql.ApplicationWebhookTypeConfigurationChanged,
 		})
 
 		require.NoError(t, err)
 		addReq := gcli.NewRequest(
 			fmt.Sprintf(`mutation {
-			result: addApplicationWebhook(applicationID: "%s", in: %s) {
+			result: addWebhook(applicationID: "%s", in: %s) {
 					%s
 				}
 			}`, actualApp.ID, webhookInStr, tc.gqlFieldsProvider.ForWebhooks()))
 		saveQueryInExamples(t, addReq.Query(), "add aplication webhook")
 
-		actualWebhook := graphql.ApplicationWebhook{}
+		actualWebhook := graphql.Webhook{}
 		err = tc.RunQuery(ctx, addReq, &actualWebhook)
 		require.NoError(t, err)
 		assert.Equal(t, "new-webhook", actualWebhook.URL)
 		assert.Equal(t, graphql.ApplicationWebhookTypeConfigurationChanged, actualWebhook.Type)
 		id := actualWebhook.ID
 		require.NotNil(t, id)
 
+		// add to non exist application
+
+		//GIVEN
+		webhookInStr, err = tc.graphqlizer.WebhookInputToGQL(&graphql.WebhookInput{
```

Comment:
IMO it will be better to have a separate test when we test all cases when we try to add a dependent object for non-existing application. Thanks to that our tests will be shorter. 
```
func TestAddDependentObjectsWhenAppDoesNotExist(t *testing.T) {
	t.Run("add webhooks", func(t *testing.T) {

	})
}
```